packages/react-native/src/primitives/Button/types.ts

```diff
@@ -1,9 +1,10 @@
-import { PressableProps, StyleProp, TextStyle } from 'react-native';
+import { PressableProps, StyleProp, TextStyle, ViewStyle } from 'react-native';
 
 export interface ButtonProps extends PressableProps {
+  style?: StyleProp<ViewStyle>;
```

Comment:
Can we leverage the built in `PressableProps['style']` type to allow customers (and us) to take advantage of passing in callback functions to programmatically apply styles in pressed/not-pressed states? I think this change may also break the `Button` styling used for In-App Messaging

Reply:
added!